.goreleaser.yml

```diff
@@ -157,7 +157,7 @@ docker_manifests:
     image_templates:
       - 'golangci/golangci-lint:v{{ .Major }}.{{ .Minor }}-alpine-amd64'
       - 'golangci/golangci-lint:v{{ .Major }}.{{ .Minor }}-alpine-arm64'
-  - name_template: 'golangci/golangci-lint:{{ .Tag }}'
+  - name_template: 'golangci/golangci-lint:{{ .Tag }}-alpine'
```

Comment:
ohhh, very difficult to review it..
maybe more whitespaces?

like
```yml
  - name_template: 'golangci/golangci-lint:latest-alpine'
    image_templates:
      - 'golangci/golangci-lint:{{ .Tag }}-alpine-amd64'
      - 'golangci/golangci-lint:{{ .Tag }}-alpine-arm64'
 
  - name_template: 'golangci/golangci-lint:v{{ .Major }}.{{ .Minor }}-alpine'
    image_templates:
      - 'golangci/golangci-lint:v{{ .Major }}.{{ .Minor }}-alpine-amd64'
      - 'golangci/golangci-lint:v{{ .Major }}.{{ .Minor }}-alpine-arm64'
  ```

Reply:
You are right, I added more whitespaces.